projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -2077,8 +2077,9 @@ public function process_submission() {
 		);
 		$footer_ip = null;
 		if ( $comment_author_ip ) {
-			$footer_ip = sprintf(
-			/* translators: Placeholder is the IP address of the person who submitted a form. */
+			$comment_author_ip = $comment_author_ip . ' ' . $response->get_country_flag();
+			$footer_ip         = sprintf(
+				/* translators: Placeholder is the IP address of the person who submitted a form. */
```

Comment:
Might consider using something like $comment_author_ip_with_flag for clarity rather than just overwriting $comment_author_ip. 

Also if get_country_flag() returns '', then I think we'll have an empty string on the back of the IP.